test/transactions/transaction.js

```diff
@@ -192,63 +196,67 @@ describe('transaction.js', () => {
 		});
 	});
 
-	describe('#createTransaction with data', () => {
+	describe('#createTransaction with data @now', () => {
 		const createTransaction = transaction.createTransaction;
 		let trs = null;
+		const feeWithData = 20000000;
+		const testAmount = 1000;
 
 		it('should create transaction with data', () => {
-			trs = createTransaction('58191285901858109L', 1000, 'secret', '', 'data');
+			trs = createTransaction(testRecipientAddress, testAmount, testSecret, '', testData);
 			(trs).should.be.ok();
-			(trs.fee).should.be.equal(20000000);
+			(trs.fee).should.be.equal(feeWithData);
 		});
 
 		it('should create transaction with invalid data', () => {
 			(() => {
-				trs = createTransaction('58191285901858109L', 1000, 'secret', '', '\xFF\xFE\x00\x00utf32le string');
-			}).should.throw('invalid encoding UTF-32LE in transaction data');
+				trs = createTransaction(testRecipientAddress, testAmount, testSecret, '', Buffer.from('hello'));
+			}).should.throw('invalid encoding in transaction data');
 		});
 	});
 
 	describe('#createTransaction with secondSignature and data', () => {
 		const createTransaction = transaction.createTransaction;
 		let trs = null;
-		const secondSecret = 'second secret';
+		const feeWithData = 20000000;
+		const testAmount = 1000;
 
 		it('should be a function', () => {
 			(createTransaction).should.be.type('function');
 		});
 
 		it('should create transaction with second signature and data', () => {
-			trs = createTransaction('58191285901858109L', 1000, 'secret', secondSecret, 'data');
+			trs = createTransaction(testRecipientAddress, testAmount, testSecret, secondSecret, testData);
 			(trs).should.be.ok();
 		});
 
 		describe('returned transaction', () => {
 			it('should contain data field with string value', () => {
 				(trs.asset.data).should.be.type('string');
-				(trs.fee).should.be.equal(20000000);
+				(trs.fee).should.be.equal(feeWithData);
 			});
 		});
 	});
 
 	describe('#createTransaction with secondSignature and data', () => {
 		const createTransaction = transaction.createTransaction;
 		let trs = null;
-		const secondSecret = 'second secret';
+		const feeWithData = 20000000;
+		const testAmount = 1000;
```

Comment:
If we're using the same default amount in different places, maybe it should go in an outer `describe` block.